index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>BLACKPINK Fansite</title>
+  <link rel="stylesheet" href="./styles/style.css">
+  <link rel="stylesheet" href="./styles/index.css">
+  <script src="https://kit.fontawesome.com/dbc83a84d0.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <header>
+    <h1>BLɅϽKPIИK</h1>
+    <nav>
+      <ul id="navlinks">
+        <li><a href="../index.html">♪ Home</a></li>
+        <li><a href="pages/facts.html">♪ Facts</a></li>
+          <li><a href="pages/gallery.html">♪ Gallery</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main class="wrapper">
+    <section class="content">
+      <img alt="Blackpink Group Picture" src="https://www.billboard.com/wp-content/uploads/2022/07/blackpink-press-billboard-1548.png?w=942&h=623&crop=1&resize=942%2C623">
```

Comment:
Totally cool in this case, but in practice you'd want to avoid using images scavenged from the web because you have no control over access. For instance, if you're depending on this image and the hosting site removes it, you now have a broken image link!